### ISSUE TITLE
Microbial Diversity and Composition in Six Different Gastrointestinal Sites among Participants Undergoing Upper Gastrointestinal Endoscopy in Henan, China

### ISSUE THREAD

**Issue by SvetlanaUP** [waldronlab/BugSigDBcuration#252]
Microbial Diversity and Composition in Six Different Gastrointestinal Sites among Participants Undergoing Upper Gastrointestinal Endoscopy in Henan, China – Minjuan Li – Microbiology Spectrum
https://journals.asm.org/doi/10.1128/spectrum.00645-21

**Comment by AleruDivine** [comment]
Thank you so much @SvetlanaUP I'll get started right away!

**Comment by AleruDivine** [comment]
Hi, @SvetlanaUP I have completed the curation of this study and would like to submit it for review.

Link to study: https://bugsigdb.org/Study_953

I have some concerns I'd like to address during office hours today. I've already posed these questions in the BugSigDB channel and received some feedback, but I'd appreciate confirmation from the mentors. 

I would like to know if I can edit the study if I receive conflicting feedback from the mentors.

Thank you so much!

Additionally, can I claim available issues to curate as I wait for this review?

**Comment by SvetlanaUP** [comment]
@AleruDivine yes, you can claim one of the available issues.

**Comment by SvetlanaUP** [comment]
As for different feedback from mentors, please get back to mentors to confirm your final correction.

**Comment by AleruDivine** [comment]
Hi @SvetlanaUP I would like to retract this study from being reviewed at this time because it is very incomplete. I have gotten a comprehensive feedback from @cmirzayi. 
This study is not nearly done as there are multiple experiments to curate.
I will take my time in completing this curation and I will let you know when it is ready for review.

Thank you so much!

**Comment by SvetlanaUP** [comment]
@AleruDivine yes, take you time! This one is indeed with a lot of experiments to curate.

**Comment by SvetlanaUP** [comment]
Is this study ready for review? @AleruDivine

**Comment by SvetlanaUP** [comment]
Please do not edit it anymore. Thanks.

**Comment by AleruDivine** [comment]
Noted! Thank you so much!

**Comment by AleruDivine** [comment]
Link to study: https://bugsigdb.org/Study_953

**Comment by AleruDivine** [comment]
Good afternoon @SvetlanaUP, 

While reviewing my previously submitted studies, I identified an issue with this particular study. Although it is not among those marked as incomplete. This study currently contains duplicate signatures due to an initial pairwise comparison rather than a one-to-all comparison. This is from misinterpretating the LEfSe results in Figure 2.

After a thorough review, I have come to realize that this study should contain 12 curated experiments, rather than the 45 currently curated. The additional curated experiments should be removed, as they involve only alpha diversity measures and no differentially abundant signatures.

**Details on 12 experiments for curation include:**

1. **Experiments 1-6**: These involve one-to-all comparisons across six gastrointestinal (GI) tract sites (Saliva, esophageal swab, cardia biopsy, noncardia biopsy, gastric juice, and fecal specimens), as shown by LEfSe differential abundance results in Figure 2b.
2. **Experiments 7-10**: The authors checked the impact of H. pylori infection on microbial characteristics in cardia and noncardia biopsy specimens, with significant differential abundance results in Table 2 for comparisons (PC/NC, PN/NN, PC/PN, NC/NN).
3. **Experiments 11-12**: The authors again checked the influence of both H. pylori infection and pH value on gastric juice microbial characteristics, with significant differential abundance results provided in Table S4. The comparisons made here are High pH (H) vs. Low pH (L) and H. pylori positive (P) vs. H. pylori negative (N).

I genuinely apologize for not catching these details initially. I believe I have gained a clearer understanding and would appreciate the chance to correct my errors.

Thank you so much!🙏

**Comment by SvetlanaUP** [comment]
@AleruDivine no need to apologize. Our curation should be sometimes rechecked and it's great that you spotted errors. Please do corrections. And please do let me know what should be deleted, Experiments 13-45?

**Comment by AleruDivine** [comment]
Thank you so much 😊 
Kindly delete the following:
1. Experiments 13 - 45
2. Signature 2 in experiments 1 - 6

**Comment by SvetlanaUP** [comment]
All deleted.

**Comment by AleruDivine** [comment]
Thank you so much Ma’am. I must apologize for the hassle. 🙏

**Comment by KateRasheed** [comment]
Good morning @SvetlanaUP . I would like to review this paper. Thank you.

**Comment by KateRasheed** [comment]
Good morning @AleruDivine . Great job on this curation. Just this correction to make on the study before we close it:

Please remove Veillonella, Neisseria, Pseudomonas, Brevudimonas, Selenomonas and Delftia from experiment 9 signature 1, they weren't significant.

Thank you so much.

**Comment by AleruDivine** [comment]
Thank you for the review @KateRasheed.
All corrections have been effected.

**Comment by KateRasheed** [comment]
Thanks for the correction. This review is complete @SvetlanaUP

**Comment by SvetlanaUP** [comment]
https://bugsigdb.org/Study_953 reviewed.
